moveos/moveos-stdlib/moveos-stdlib/sources/context.move

```diff
@@ -142,6 +143,12 @@ module moveos_std::context {
         obj_ref
     }
 
+    #[private_generics(T)]
+    public fun new_singleton_object<T: key>(self: &mut Context, value: T): ObjectRef<T> {
+        let object_id = object::singleton_object_id<T>();
```

Comment:
Since it is a singleton object, do we need to check whether the object exists?

Reply:
> Since it is a singleton object, do we need to check whether the object exists?

Because the ObjectID will be the same, new should not appear the second one, and better error information can be added.